src/App.js

```diff
@@ -1,21 +1,89 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
+import React from 'react';
+import {BrowserRouter, Route} from 'react-router-dom'
+import {addIndex, append, curry, evolve, inc, map, pipe, omit, when} from "ramda";
+
+import Toolbar from './Toolbar/Toolbar';
+import Home from './Home/Home';
+import MealDetail from '@/MealDetail/MealDetail';
+import {sortById} from '@/business/meals';
+import MealService from '@/services/meals';
+
+import './App.less';
+
+
+const identifyMeal = (meal, id) => Object.assign({id}, meal);
+
+const unIdentifyMeal = (meal) => omit("id", meal);
+
+const mapWithIndex = addIndex(map);
+
+const useMeal = evolve({
+	lastUsed: () => Date.now(),
```

Comment:
yeah, I think so. the evolve function will pass the value of the original object's property here, but since now doesn't take a param, it should just ignore it.

Reply:
Ahh - that's a good point. You're explicitly throwing away params here, which may actually be quite useful - since `Date.now(params)` is liable to blow up.

Reply:
ps this evolve function is pretty neat. turns out ramda has some really, really, really cool tricks up its sleeve.

Reply:
Oh wow - yeah - I hadn't read the doc until just now for it. That is a really slick bit of code.